connexion/decorators/produces.py

```diff
@@ -57,17 +57,18 @@ def get_data_status_code(data):
         """
         url = flask.request.url
         logger.debug('Getting data and status code', extra={'data': data, 'data_type': type(data), 'url': url})
+        status_code, headers = 200, []
```

Comment:
For the default, empty, value it can be just an empty dict. I'll add the recommendation to use an ordered dict in the case the order matters as part of #78.